boards/ti/sk_am62/sk_am62_am6234_a53.dts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2023 Enphase Energy
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <ti/ti_am6234_a53.dtsi>
+
+/ {
+	model = "TI AM62X STARTER KIT (SK) EVALUATION MODULE (EVM)";
+	compatible = "ti,am62x_a53_sk";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,sram = &ddr0;
+	};
+
+	cpus {
+		cpu@0 {
+			status = "okay";
+		};
+		cpu@1 {
+			status = "disabled";
+		};
+		cpu@2 {
+			status = "disabled";
+		};
+		cpu@3 {
+			status = "disabled";
+		};
+	};
+
+	ddr0: memory@82000000 {
+		/* Note: This board actually has 2GB DRAM available */
+		reg = <0x82000000 DT_SIZE_M(1)>;
+	};
+};
+
+&pinctrl {
```

Comment:
Move pinctrl in ``sk-am62_am6234_a53-pinctrl.dts``. I will update the phyBOARD-Lyra soon too.